modules/go-compatibility-and-support-matrix.adoc

```diff
@@ -14,7 +14,8 @@ In the table, features are marked with the following statuses:
 |===
 |*OpenShift GitOps* 8+|*Component Versions*|*OpenShift Versions*
 
-|*Version*|*kam*    |*Helm*  |*Kustomize*|*Argo CD*|*ApplicationSet*|*Dex*     |*RH SSO* |**Notifications Controller**|
+|*Version* |*kam*    |*Helm*  |*Kustomize* |*Argo CD*|*ApplicationSet* |*Dex*     |*RH SSO* |**Notifications Controller**|
```

Comment:
Change `**Notifications Controller**` to `*Notifications Controller*`. Two reasons:

- AsciiDoc does not need double asterisks to render bold font.
- Single asterisks will maintain consistency with the rest of the table headers.